README.md

```diff
@@ -1,11 +1,14 @@
 # Metal
 
-A programming language. We'll add a catchy slogan soon, pinky swear
+Prototype compiled language built in Rust with LLVM. We'll add a catchy slogan later.
```

Comment:
If you feel like it, feel free to remove the latter sentence, no need to make compromises here :D